tests/test_utils.py

```diff
@@ -133,6 +133,55 @@ def test_get_editor_precedence():
     assert utils.get_editor(onyo_root) == 'third'
 
 
+def test_create_faux():
+    """
+    Test the creation of unique serial numbers with the dafault length through
+    calling the create_faux function multiple times and comparing the generated
+    faux.
+    """
+    onyo_root = './'
+    faux = []
+    number_of_faux = range(0, 100)
+
+    # technically, creating multiple faux serial numbers without actually
+    # creating assets might lead to duplicates. In practice, this should not
+    # happen if they are sufficiently long and random
+    for i in number_of_faux:
+        new_faux = utils.create_faux(utils.get_list_of_assets(onyo_root))
+        # make sure that the faux serial is unique
+        assert new_faux not in faux
+        faux.append(new_faux)
+
+    # check that the correct number of unique faux serials was created
+    assert len(faux) == len(number_of_faux)
+
+
+def test_create_faux_different_length():
+    """
+    Create many faux with different length. Because of the different length,
```

Comment:
"Faux" is not a noun. Many faux(fake) serials are generated.

Length -> lengths.

Is there a max length? If so, that should be documented and checked for in the `create_faux()` function.